app/models/openshift_container_image.rb

```diff
@@ -0,0 +1,13 @@
+class OpenshiftContainerImage < ContainerImage
```

Comment:
This seems openshift specific and probably should go into the openshift provider gem itself.